ai-cfia-ia-acia/docs/about/team/internships.md

```diff
@@ -2,4 +2,48 @@
 sidebar_position: 4
 ---
 
-# Internships with the AI Lab
\ No newline at end of file
+# Internships at the AI Lab
+
+### [Allagbe, Kotchikpa Guy-Landry](https://www.linkedin.com/in/guy-landry-allagbe/) - Backend Automne 2023
+> J'ai choisi ce stage parce qu'il s'aligne parfaitement avec les compétences que je possède déjà et que je souhaite développer davantage. Ce stage représente une opportunité idéale pour moi de grandir professionnellement tout en contribuant à un domaine qui me passionne.
+
+> Pour améliorer l'intégration des futurs stagiaires, le superviseur nous a demandé de contribuer à la rédaction des guides d'accueil, ce qui a renforcé l'esprit d'équipe. J'ai été impliqué dans les réunions d'équipe et j'ai même eu l'occasion de rencontrer la haute direction. 
+
+> Malgré des débuts un peu chaotiques, je suis globalement satisfait de mon intégration et je suis très satisfait de mon expérience de stage jusqu'à présent. Mon superviseur est non seulement compétent, mais il me donne aussi la confiance nécessaire pour atteindre les objectifs que nous avons fixés.  Cela ajoute une couche de motivation et rend l'expérience d'autant plus gratifiante.
+
+### [Fressard, Mélanie](https://www.linkedin.com/in/melanie-fressard/) - Base de données Automne 2023
+> Puisque j'étais en compétition avec mon club étudiant, je n'ai malheureusement pas commencé à travailler en même temps que les autres stagiaires mais j'ai tout de même reçu un accueil vraiment chaleureux. L'intégration prend en compte toutes les formations nécessaires en matière de sécurité de l'information mais aussi du côté technique selon notre rôle. 
+
+> Le fait que nous utilisons des outils qui encouragent la cohésion d’équipe et facilitent la collaboration entre tout le monde permet d’avancer plus efficacement dans les projets, de se sentir inclus et d’avoir l’avis de chacun sur les décisions importantes.
+
+
+### [Lemire, William](https://www.linkedin.com/in/wlemire/) - Backend Automne 2023
+> Je suis retourné aux études pour éventuellement travailler dans le domaine de l'intelligence artificielle. Je ne suis pas tant intéressé de développer l'IA en tant que telle, mais plutôt de l'utiliser et de l'adapter à des contextes pratiques pour résoudre des problèmes réels et c'est exactement ce que ce stage me permet de réaliser. J'ai également l'occasion de faire valoir mon expérience antérieure en relations publiques/communications en travaillant sur le projet de site web, ce qui m'est fort intéressant.
+
+> Le processus d'intégration m'a semblé décousu, comme dans une start-up. Je m'attendais à quelque chose de plus structuré, étant un poste au gouvernement fédéral. En fait, ce n'est pas mal structuré, c'est en construction et nous avons tous notre part à jouer pour améliorer le processus. 
```

Comment:
J'enlèverais ce bloc de texte, c'était plutôt destiné à la coordonnatrice de stage.

Reply:
This block of text ↓
"Le processus d'intégration m'a semblé décousu, comme dans une start-up. Je m'attendais à quelque chose de plus structuré, étant un poste au gouvernement fédéral. En fait, ce n'est pas mal structuré, c'est en construction et nous avons tous notre part à jouer pour améliorer le processus."